opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -236,6 +242,12 @@ message Span {
     // dropped_attributes_count is the number of dropped attributes. If the value is 0,
     // then no attributes were dropped.
     uint32 dropped_attributes_count = 5;
+
+    // Flags from the W3C traceparent, if provided.
```

Comment:
Nit: Since these are the flags from the "linking" span's context, I wonder if this comment should say something like "Flags from the linking span's trace context, if provided.".

Reply:
Fixed.